### ISSUE TITLE
Warn Everytime someone leaves

### ISSUE THREAD

**Issue by ZDaPlaY** [IntellectualSites/FastAsyncWorldEdit#1205]
### Server Implementation

Purpur

### Server Version

1.16.5

### Describe the bug

every time someone comes out this comes out, any way to fix it?

[14:27:37 WARN]: java.lang.reflect.InaccessibleObjectException: Unable to make field java.lang.ThreadLocal$ThreadLocalMap java.lang.Thread.threadLocals accessible: module java.base does not "opens java.lang" to unnamed module @6150c062
[14:27:37 WARN]:        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357)
[14:27:37 WARN]:        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
[14:27:37 WARN]:        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:177)
[14:27:37 WARN]:        at java.base/java.lang.reflect.Field.setAccessible(Field.java:171)
[14:27:37 WARN]:        at com.boydti.fawe.object.collection.CleanableThreadLocal.iterate(CleanableThreadLocal.java:94)
[14:27:37 WARN]:        at com.boydti.fawe.object.collection.CleanableThreadLocal.getAll(CleanableThreadLocal.java:61)
[14:27:37 WARN]:        at com.sk89q.worldedit.session.request.Request.getAll(Request.java:55)
[14:27:37 WARN]:        at com.sk89q.worldedit.extension.platform.Actor.cancel(Actor.java:235)
[14:27:37 WARN]:        at com.sk89q.worldedit.entity.Player.unregister(Player.java:403)
[14:27:37 WARN]:        at com.sk89q.worldedit.bukkit.BukkitPlayer.unregister(BukkitPlayer.java:417)
[14:27:37 WARN]:        at com.boydti.fawe.bukkit.FaweBukkit.onPlayerQuit(FaweBukkit.java:267)
[14:27:37 WARN]:        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1655.execute(Unknown Source)
[14:27:37 WARN]:        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
[14:27:37 WARN]:        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[14:27:37 WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[14:27:37 WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.PlayerList.disconnect(PlayerList.java:598)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1977)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.NetworkManager.handleDisconnection(NetworkManager.java:713)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.ServerConnection.c(ServerConnection.java:193)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1596)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:451)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1415)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1141)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:292)
[14:27:37 WARN]:        at java.base/java.lang.Thread.run(Thread.java:831)

### To Reproduce

every time someone comes out this comes out, any way to fix it?

### Expected behaviour

every time someone comes out this comes out, any way to fix it?

[14:27:37 WARN]: java.lang.reflect.InaccessibleObjectException: Unable to make field java.lang.ThreadLocal$ThreadLocalMap java.lang.Thread.threadLocals accessible: module java.base does not "opens java.lang" to unnamed module @6150c062
[14:27:37 WARN]:        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357)
[14:27:37 WARN]:        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
[14:27:37 WARN]:        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:177)
[14:27:37 WARN]:        at java.base/java.lang.reflect.Field.setAccessible(Field.java:171)
[14:27:37 WARN]:        at com.boydti.fawe.object.collection.CleanableThreadLocal.iterate(CleanableThreadLocal.java:94)
[14:27:37 WARN]:        at com.boydti.fawe.object.collection.CleanableThreadLocal.getAll(CleanableThreadLocal.java:61)
[14:27:37 WARN]:        at com.sk89q.worldedit.session.request.Request.getAll(Request.java:55)
[14:27:37 WARN]:        at com.sk89q.worldedit.extension.platform.Actor.cancel(Actor.java:235)
[14:27:37 WARN]:        at com.sk89q.worldedit.entity.Player.unregister(Player.java:403)
[14:27:37 WARN]:        at com.sk89q.worldedit.bukkit.BukkitPlayer.unregister(BukkitPlayer.java:417)
[14:27:37 WARN]:        at com.boydti.fawe.bukkit.FaweBukkit.onPlayerQuit(FaweBukkit.java:267)
[14:27:37 WARN]:        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1655.execute(Unknown Source)
[14:27:37 WARN]:        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
[14:27:37 WARN]:        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[14:27:37 WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[14:27:37 WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.PlayerList.disconnect(PlayerList.java:598)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1977)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.NetworkManager.handleDisconnection(NetworkManager.java:713)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.ServerConnection.c(ServerConnection.java:193)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1596)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:451)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1415)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1141)
[14:27:37 WARN]:        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:292)
[14:27:37 WARN]:        at java.base/java.lang.Thread.run(Thread.java:831)

### Screenshots / Videos

![image](https://user-images.githubusercontent.com/88196950/127660645-3df130de-a96c-4e7e-9f7a-c69271a5afc3.png)

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/a8dd82fa66164b58a7e2d059f90a6b8b

### Fawe Version

1.16-695;04610821

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit-1.17/ and the issue still persists.

**Comment by PierreSchwang** [comment]
![grafik](https://user-images.githubusercontent.com/27054324/127661424-e2701b2c-ade8-49af-b4b1-331367583e64.png)

**Comment by NotMyFault** [comment]
tldr; actually go thru the checklist before ticking it.
